### PR TITLE
:pencil:  updating the website tagline, the hook

### PR DESCRIPTION
**Pull request by MatissJanis**
Updated the tagline (the hook) for the website.

For people coming in for the first time - they don't know (and care) about the historical context of open source actual vs closed source. So slightly changed the copy to focus on the truly important things - the features.

### CODE REVIEW

File: .editorconfig
Comment (j-f1) [inline .editorconfig:1]: :100:

File: src/pages/index.js [hunk 1]
```diff
@@ -13,17 +13,16 @@ export default function Hello() {
...
       <div class={`${classes.main} ${classes.container}`}>
-        <h1>The same Actual Budget, but Open Source</h1>
+        <h1>Enjoy managing your finances</h1>
         <p class={classes.heroText}>
-          This website is for the Open Source version of Actual Budget, it is a
-          super fast privacy-focused app for managing your finances. You own
-          your data, and we will sync it across all devices with optional
-          end-to-end encryption.
+          Actual Budget is a super fast privacy-focused app for managing your
+          finances. You own your data and can do with it whatever you want.
```
Comment (j-f1) [inline src/pages/index.js:19]: This reads slightly better IMO

```suggestion
          finances. You own your data and can do whatever you want with it.
```

**Comment by MatissJanis** [comment]
Let me know what you think.

IMO this is the most important piece of text we have on the page and thus it deserves extra scrutiny. Please tear this to PR to shreds (but suggest alternatives). Lets come up with the best possible tagline and description.

**Review by j-f1 — APPROVED** [review]

**Comment by j-f1** [comment]
I think this is a great overview!

File: src/pages/index.js (hunk 1 above)
Comment (MatissJanis) [reply src/pages/index.js]: Agreed! Great to have native-speakers on the team.

**Comment by YagiOoda** [comment]
I may suggest adding "zero-based budget" or "envelope budget". 
Maybe "Actual Budget is a super fast privacy-focused app for managing your budget in a zero-based way."
Or "Actual Budget is a super fast privacy-focused zero-based budgeting app for managing your finances."

Not sure if this should go in the tagline or the description, though.

**Comment by Kidglove57** [comment]
I think we definitely need a reference here to the budgeting methodology. I found Actual back in 2019 solely because I was looking for an alternative to the likes of YNAB, Moneywell or Goodbudget. I was very specific about wanting an envelope budgeting solution.

Having said that I'm not sure that the words "Zero Based" (or similar) are as well known as "Envelope Budgeting" (unless perhaps to ex YNAB users).

I would favour something like the following:

_"Your Finances made simple.

Actual Budget is a super fast and privacy-focused app for managing your finances. It is based around the well proven and much loved Envelope Budgeting methodology.

You own your data and can do whatever you want with it. Actual features multi-device sync, with optional end-to-end encryption"_

**Comment by rich-howell** [comment]
As a quick update, I won't be renewing the .org domain next year, I would begin planning whatever change that may be required before then.

**Comment by MatissJanis** [comment]
Updated:
Screenshot 2023-06-23 at 08 26 51

**Pull request merged**